packages/vite/src/node/plugins/css.ts

```diff
@@ -565,7 +565,8 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
             type: 'asset',
             source: chunkCSS,
           })
-          const originalName = isPreProcessor(lang) ? cssAssetName : cssFileName
+          const originalName =
+            isPreProcessor(lang) || lang === 'pcss' ? cssAssetName : cssFileName
```

Comment:
It seems like we could use `cssAssetName` directly, and remove the `originalName` variable. It would also cover cases like `style.postcss` and `style.sss` 🤔 

So removing these two lines, and update line 573 to:

```js
.set(referenceId, { originalName: cssAssetName, isEntry })
```

should do the trick.